acquisitions/src/main/resources/thunderjet/cross-modules/features/check-paymentstatus-after-reopen.feature

```diff
@@ -21,7 +21,7 @@ Feature: Check paymentStatus after reopen
     * def createOrder = read('classpath:thunderjet/mod-orders/reusable/create-order.feature')
```

Comment:
Here you can also remove few reusable features that are already globally present

Reply:
There are lots of features that could benefit from this, probably better to have some PR to tend to such changes in our tests